modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/buttons/RowOutlinedButton.kt

```diff
@@ -46,42 +57,62 @@ fun RowOutlinedButton(
             shape = RoundedCornerShape(12.dp),
             border = border,
             colors = colors,
-            modifier = Modifier
-                .fillMaxWidth()
+            modifier = Modifier.fillMaxWidth()
         ) {
-            iconImage?.let {
-                Icon(imageVector = it, contentDescription = "")
+            Box(Modifier.fillMaxWidth(), contentAlignment = Alignment.CenterStart) {
+                RowOutlinedImage(
+                    image = leadingIcon,
+                    colors = colors,
+                    tintIcon = tintIcon
+                )
+                Row(
+                    horizontalArrangement = Arrangement.Center,
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    RowOutlinedImage(
+                        image = textIcon,
+                        colors = colors,
+                        tintIcon = tintIcon
+                    )
+                    TextH30(
+                        text = text,
+                        color = colors.contentColor(enabled = true).value,
```

Comment:
I don't think we want the left text alignment default here, although it only becomes an issue when the text wraps.

```suggestion
                        color = colors.contentColor(enabled = true).value,
                        textAlign = TextAlign.Center,
```